ads/google/doubleclick.js

```diff
@@ -28,7 +28,10 @@ export function doubleclick(global, data) {
     )['attributes']['_context'];
     // Make this easy to rollback in case of emergency.
     if (context['experimentToggles'][`rollback-dfd-${data.type}`]) {
-      return deprecatedDoubleclick(window, data);
+      if (data['type'] != 'doubleclick' &&
```

Comment:
Should be an OR.  Idea is that doubleclick should always be allowed to used deprecated as well as any non-doubleclick that is not in the experiment